### PR TITLE
Update policy parameter with 2019 IRS final values 

### PR DESCRIPTION
**Pull request by Peter-Metz**
This PR updates `policy_current_law.json` and associated test files to reflect final IRS values for 2019. Values for indexable parameters 2020 and beyond were calculated with `ppp.py`. Note that Tax-Data has not yet been updated to reflect the 2020 CBO inflation assumptions, so `ppp.py` will have to be run again once Tax-Data (and `growfactors.csv`) is updated.

Relevant IRS forms:

[f1040.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221209/f1040.pdf)
[f1040sa.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221211/f1040sa.pdf)
[i1040gi.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221217/i1040gi.pdf)
[i1040s8.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221218/i1040s8.pdf)
[i1040sca.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221219/i1040sca.pdf)
[f6251.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221221/f6251.pdf)
[f8863.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221222/f8863.pdf)
[f8995a.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221223/f8995a.pdf)
[f461.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221228/f461.pdf)
[iw2w3.pdf](https://github.com/PSLmodels/Tax-Calculator/files/4221235/iw2w3.pdf)

cc: @MattHJensen

### CODE REVIEW

**Comment by MattHJensen** [comment]
@Peter-Metz, thanks a lot for opening this PR. I will review this week.

**Comment by Peter-Metz** [comment]
@MattHJensen and I had a question regarding changes to the reform output files (e.g. `2017_law.out.csv`) so we thought we'd document the history of thought here:

We were wondering why some reform output test files include changes to most/all output variables (e.g. `Larson2019.out.csv`) while others only include changes to `payrolltax` (e.g. `Renacci.out.csv`)?

The reason for this is that `Larson2019.json` uses current law as the baseline while `Renacci.json` uses `2017_law.json` as its baseline. Since the modifications in this PR relate to `policy_current_law.json`, they do not effect most of the parameters in `2017_law.json`. The exception to this is `SS_earnings_c`, which does not appear in `2017_law.json`. For this reason, reforms using 2017 law as their baseline will still see a change in `payrolltax` whenever `SS_earnings_c` is changed.

File: taxcalc/policy_current_law.json [hunk 1]
```diff
@@ -745,14 +753,14 @@
                   [9e99, 9e99, 9e99, 9e99, 9e99],
                   [9e99, 9e99, 9e99, 9e99, 9e99],
                   [250000.0, 500000.0, 250000.0, 250000.0, 500000.0],
-                  [255600.0, 511200.0, 255600.0, 255600.0, 511200.0],
```
Comment (MattHJensen) [inline taxcalc/policy_current_law.json]: @Peter-Metz, do you know why the 2019 values here appeared to be rounded already?

**Comment by MattHJensen** [comment]
> The exception to this is SS_earnings_c, which does not appear in 2017_law.json. For this reason, reforms using 2017 law as their baseline will still see a change in payrolltax whenever SS_earnings_c is changed.

@Peter-Metz and my plan is to review whether `SS_earnings_c` or any other parameters ought to be included in `2017_law.json` in a PR that follows this one. So the reform tests results may change again in that PR.

File: taxcalc/policy_current_law.json (hunk 1 above)
Comment (MattHJensen) [reply taxcalc/policy_current_law.json]: Peter confirmed over chat that the previous years are correct, so I am going to leave this.

**Comment by MattHJensen** [comment]
@Peter-Metz, thanks again for this PR. Merging.

**Pull request merged**